src/api/index.js

```diff
@@ -128,9 +128,9 @@ export function getTaskRunLog(name, namespace) {
   return get(uri);
 }
 
-export function createPipelineRun(payload) {
-  const uri = `${apiRoot}/`;
-  return post(uri, payload);
+export function createPipelineRun({ id, ...rest }, namespace) {
```

Comment:
I don't see where the `id` would be provided, this should probably be removed.

There's also no need to split the params, you're not doing anything with them individually here, you're just passing them all through so you can do it directly:

```js
export function createPipelineRun(params, namespace) {
  const uri = getAPI('pipelineruns', { namespace });
  return post(uri, params);
}
```